capture/win32.js

```diff
@@ -1,18 +1,19 @@
 module.exports = function(options, callback) {
+  var fs = require("fs");
+  var childProcess = require("child_process");
+  var path = require("path");
 
-	var fs = require('fs');
-	var childProcess = require('child_process');
-	var path = require('path');
-	
-	var nircmd = childProcess.spawn(path.join(__dirname, "bin", "nircmd.exe"), ["savescreenshot", options.output]);	
+  var nircmd = childProcess.spawn(path.join(__dirname, "bin", "nircmd.exe"), [
+    "savescreenshotfull",
```

Comment:
I think it might be a better idea to use an option for the full screen. This most probably would break previous implementations.